_config.yml

```diff
@@ -49,14 +49,14 @@ contentful:
     query: fields.category=common
     map:
       content_block: content
-  page:
-    query: "fields.collections[exists]=false"
```

Comment:
Was this intentional?

Reply:
I was having issues with builds/contentful because collections was removed from pages in development. This has since been reverted so I will in turn revert this.

Reply:
Not important, but now it just looks like you added some extra spaces.